### PR TITLE
Pass latest or stable to build/push-official-release.sh.

### PR DESCRIPTION
**Pull request by david-mcmahon**

### CODE REVIEW

File: build/push-official-release.sh [hunk 1]
```diff
@@ -21,11 +21,12 @@ set -o nounset
 set -o pipefail
 
 if [ "$#" -ne 1 ]; then
```
Comment (ihmccreery) [inline build/push-official-release.sh]: `-ne 2`?

File: release/build-official-release.sh [hunk 2]
```diff
@@ -59,6 +59,12 @@ else
   RELEASE_BRANCH="release-${VERSION_MAJOR}.${VERSION_MINOR}"
 fi
 
+if [[ "$KUBE_RELEASE_VERSION" =~ alpha|beta ]]; then
+  RELEASE_TXT_FILE="latest"
```
Comment (ihmccreery) [inline release/build-official-release.sh]: `KUBE_RELEASE_TYPE`, for consistency above?

File: build/push-official-release.sh (hunk 1 above)
Comment (ihmccreery) [inline build/push-official-release.sh]: Except you handle the single-arg case, too.  So I guess `< 1 || > 2?  Also you should make it clear in Usage that the second argument is optional.

**Comment by ihmccreery** [comment]
Looks reasonable to me; please address comments.

**Comment by david-mcmahon** [comment]
PTAL

File: build/push-official-release.sh (hunk 1 above)
Comment (ihmccreery) [inline build/push-official-release.sh]: This still won't work, because you didn't change `-ne 1`.

Comment (ihmccreery) [inline build/push-official-release.sh]: Unless I'm confused?

Comment (david-mcmahon) [inline build/push-official-release.sh]: No, you weren't confused.  PTAL.

**Comment by ihmccreery** [comment]
Sorry, forgot about this earlier.  You probably want to remove `mark-stable-release.sh` and the line in `releasing.md` that asks you to call it, right?  Since this now takes care of it for us?

**Comment by david-mcmahon** [comment]
PTAL

**Comment by ihmccreery** [comment]
LGTM; thanks David.  This looks way better than what we had before.  Yay for deleting code!

**Comment by david-mcmahon** [comment]
Any reason not to force merge this?

**Comment by ihmccreery** [comment]
Not as far as I'm concerned.  @k8s-oncall; please consider merging by hand.

**Pull request merged**